src/Common/Commands.Common/AzureDataCmdlet.cs

```diff
@@ -111,48 +111,17 @@ protected override void SaveDataCollectionProfile()
             WriteWarning(string.Format(Resources.DataCollectionSaveFileInformation, fileFullPath));
         }
 
-        protected override void PromptForDataCollectionProfileIfNotExists()
+        protected override void SetDataCollectionProfileIfNotExists()
         {
-            // Initialize it from the environment variable or profile file.
             InitializeDataCollectionProfile();
 
-            if (!_dataCollectionProfile.EnableAzureDataCollection.HasValue && CheckIfInteractive())
```

Comment:
I say we leave this out of this release.  This is not a _huge_ rush, and I do not want to skirt testing guidelines.